src/overlays/actors/ovl_En_Toryo/z_en_toryo.c

```diff
@@ -29,27 +30,343 @@ const ActorInit En_Toryo_InitVars = {
     (ActorFunc)EnToryo_Update,
     (ActorFunc)EnToryo_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Toryo/EnToryo_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Toryo/EnToryo_Destroy.s")
+static ColliderCylinderInit sColliderCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 18, 63, 0, { 0, 0, 0 } },
+};
+
+static CollisionCheckInfoInit2 sCollisionCheckInfoInit2 = { 0x00, 0x00, 0x00, 0x00, 0xFF };
+
+static DamageTable sDamageTable = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+
+typedef struct {
+    AnimationHeader* anim;
+    f32 unk_4;
+    u8 mode;
+    f32 transitionRate;
+} EnToryoAnimation;
+static EnToryoAnimation sEnToryoAnimation = { 0x06000E50, 1.0f, 0, 0 };
+
+static Vec3f sMultVec = { 800.0f, 1000.0f, 0.0f };
+
+extern FlexSkeletonHeader D_06007150;
+
+void EnToryo_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnToryo* this = THIS;
+    s32 pad;
+
+    switch (globalCtx->sceneNum) {
+        case SCENE_SPOT09:
+            if (LINK_AGE_IN_YEARS == YEARS_ADULT) {
+                this->stateFlags |= 0x1;
```

Comment:
It's not important, but we don't usually include the 0x in flags that are less than 10